### PR TITLE
add gangstergorillaz

### PR DESCRIPTION
**Pull request by Scale0**
Add ganstergorillaz

### CODE REVIEW

**Review by XJWX89 — DISMISSED** [review]

**Comment by Murtagh300** [comment]
Mint starts 30th of August 👀 @XJWX89 Can we get merged before mint goes live on 10 AM UTC please. Love you guys ❤️

File: apps/io.gangstergorillaz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Gangster Gorillaz.",
+  "href": "https://gangstergorillaz.io/",
+  "desc": "Gangster Gorillaz NFT. Not afraid to get their hands dirty. 5000 Genesis Gangster Gorillaz Coming to take VeFam on a journey through BlockChain Land. To eventually land in Our "Banana-Verse" All together.",
```
Comment (libotony) [inline apps/io.gangstergorillaz/manifest.json:4]: should be `'Banana-Verse'`

**Review by libotony — CHANGES_REQUESTED** [review]

File: apps/io.gangstergorillaz/manifest.json (hunk 1 above)
Comment (Scale0) [reply apps/io.gangstergorillaz/manifest.json]: Updated the PR.

Comment (Murtagh300) [reply apps/io.gangstergorillaz/manifest.json]: Sorry @libotony Hope you guys still around 💯

**Review by libotony — APPROVED** [review]

**Pull request merged**